clients/src/main/java/org/apache/kafka/common/requests/FindCoordinatorResponse.java

```diff
@@ -95,4 +96,21 @@ public static FindCoordinatorResponse prepareResponse(Errors error, Node node) {
             .setPort(node.port());
         return new FindCoordinatorResponse(data);
     }
+
+    public static FindCoordinatorResponse prepareResponse(Errors error, String key, Node node) {
+        FindCoordinatorResponseData data = new FindCoordinatorResponseData();
+        data.setCoordinators(Collections.singletonList(
+                new FindCoordinatorResponseData.Coordinator()
+                .setErrorCode(error.code())
+                .setErrorMessage(error.message())
+                .setKey(key)
+                .setHost(node.host())
+                .setPort(node.port())
+                .setNodeId(node.id())));
+        return new FindCoordinatorResponse(data);
+    }
+
+    public static boolean isBatch(RequestHeader header) {
+        return header.apiVersion() >= 4;
+    }
```

Comment:
This doesn't seem to be used anywhere, is this supposed to be there for convenience for the future?

Reply:
Right, deleted